### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Various README improvements, see the commit messages for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# <img alt="ORT Server" src="assets/logo.svg" width="10%"> ORT Server 
+# <img alt="ORT Server" src="assets/logo.svg" width="10%"> Eclipse Apoapsis - ORT Server 
 
-The ORT Server is a standalone application to deploy the
-[OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) as a service in the cloud.
+The [Eclipse Apoapsis](https://projects.eclipse.org/projects/technology.apoapsis) ORT Server is a standalone application
```
Comment (sschuberth) [inline README.md:3]: Maybe: "The ORT Server of the Eclipse Apoapsis project..."?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,17 @@ to deploy the [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) as
 > 
 > <img alt="Eclipse Incubation" src="https://projects.eclipse.org/modules/custom/eclipsefdn/eclipsefdn_projects/images/project_state/incubating.png" width="10%">
 
+## Community
+
+To communicate with the developers, you can:
+* Join the [Matrix chat](https://matrix.to/#/#apoapsis:matrix.eclipse.org).
+* Start a GitHub [discussion](https://github.com/eclipse-apoapsis/ort-server/discussions).
+* Join the [mailing list](https://accounts.eclipse.org/mailing-list/apoapsis-dev).
+
```
Comment (sschuberth) [inline README.md:19]: Should the #ort-server channel in ORT's Slack workspace be added as well for completeness? Or are you planning to deprecated it in favor of the Matrix chat?

Comment (mnonnenmacher) [reply README.md:19]: Yes, I have announced it now in the channel, I didn't find the time earlier.

File: README.md (hunk 1 above)
Comment (mnonnenmacher) [reply README.md]: I have used the slightly different "The Eclipse Apoapsis project's ORT Server".

**Review by sschuberth — APPROVED** [review]

**Pull request merged**